.pre-commit-config.yaml

```diff
@@ -7,10 +7,10 @@ repos:
       args:
         - --line-length=79
         - --experimental-string-processing
--   repo: https://gitlab.com/pycqa/flake8
-    rev: 3.9.2
-    hooks:
-    - id: flake8
+# -   repo: https://gitlab.com/pycqa/flake8
+#     rev: 3.9.2
+#     hooks:
+#     - id: flake8
```

Comment:
why was this commented out? Can we revert this?

Reply:
done!